backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -0,0 +1,202 @@
+from django.http import JsonResponse
+from django.views import View
+from kernelCI_app.utils import extract_error_message, extract_platform
+from django.db import connection
+
+
+class TreeDetailsSlow(View):
+    def get(self, request, commit_hash: str | None):
```

Comment:
we are missing the filters, we want to get them in this PR or will it be a regression, we can do in a separated PR but targeting this PR and move this to draft